src/utils/nostr.ts

```diff
@@ -97,22 +100,29 @@ export const sendFormResponse = async (
   }
   publicKey = publicKey || newPk;
   ciphertext = ciphertext || (await nip04.encrypt(newSk, npub, message));
-  let kind4Event = {
+  let baseKind4Event = {
```

Comment:
why?

Reply:
These 2 are different objects. One is a partial event and another is the final event. Typescript will throw an error if we try to modify an object by adding an arbitrary attribute.

If you try to define that attribute beforehand, then Typescript will throw an error while defining the object. Its anyways better to use another variable as the both of them serve different functions